app/platform_api/schema/magic_link.rb

```diff
@@ -0,0 +1,19 @@
+class Schema::MagicLink < Lockstep::ApiRecord
```

Comment:
Schema wont be required. Schema is autogenerated from swagger. Since this isn't available in swagger, we need not add it manually